boot/freeldr/freeldr/include/arch/arm/hardware.h

```diff
@@ -34,6 +34,7 @@ extern ULONG gDiskReadBuffer, gFileSysBuffer;
 
 #define DriveMapGetBiosDriveNumber(DeviceName) 0
 
+DECLSPEC_NORETURN
 FORCEINLINE VOID Reboot(VOID)
```

Comment:
(Note that this prototype differs from the other one used throughout freeldr's code.)